projects/core/components/dialog/dialog.style.less

```diff
@@ -68,6 +80,12 @@
 
     @media @mobile {
         &[data-size] {
+            .dialog-host {
```

Comment:
I think we can move this to its own `@media @mobile` and out of the host for specificity reasons too.

Reply:
OK .. I hope I put in the correct spot 😋